### DISCUSSION TITLE
How to enable "Write to the single file" when exporting data?

### DISCUSSION THREAD

**Discussion by XW-Jason** [dbeaver/dbeaver#19926] [Q&A]
I tried different selections of Output setting, but "Write to the single file" still disabled:
![image](https://github.com/dbeaver/dbeaver/assets/129360413/d65ee3ac-62b8-4dc3-bf99-327ada70ef8d)

My export target is SQL file, I need to transfer data between different database type(oracle to sqlserver)
![image](https://github.com/dbeaver/dbeaver/assets/129360413/e3753aab-1a76-492b-acdb-033b6fe0c9cd)

There are so many export files, I need write into one file to import data by SQLs.
![image](https://github.com/dbeaver/dbeaver/assets/129360413/0c49232c-039e-4d06-9ff5-55bd96856ee0)

Appreciate for support.

**Comment by LonwoLonwo** [comment]
This setting is available when you are exporting more than one table. In this case, you can export a few tables in one target file.

**Comment by peacefulbyte** [comment]
Set the maximum threads to 1
